### ISSUE TITLE
Two copies of BeautifulSoup.py

### ISSUE THREAD

**Issue by quozl** [sugarlabs/infoslicer#29]
We have two copies of BeautifulSoup.py in the source repository, of different versions.

**Comment by Aniket21mathur** [comment]
What I can conclude by analysis of the source code is that we are using some files from the old version which in turn have its own BeautifulSoup file. For proper working of the activity both the files are of significance for us. This issue points to the problem of having two copies and can be fixed by figuring out a way to merge the two files so that we are left with one. Am I right?

**Comment by quozl** [comment]
Yes.  Please propose ways to do this.  The ways I can think of are;
* remove one file and replace it with a symbolic link to the other,
* change the imports so that a single file is used,
* remove the file and require the installation of a dependency on the system, or;
* switch to using `import requests`.

(Although the last two would break compatibility with OLPC OS on XO-1 through to XO-4, which use Fedora 18, this is minor given the activity once ported to Python 3 won't run on those systems anyway.  Someone will be maintaining a branch for Fedora 18 regardless.)

**Comment by chimosky** [comment]
In addition to what @quozl said, whoever decides to fix this should consider using distro specific `BeautifulSoup`.

**Comment by quozl** [comment]
@quozl wrote:
> * remove the file and require the installation of a dependency on the system, or;

@chimosky wrote:
> consider using distro specific `BeautifulSoup`.

I'm missing something.  How is that different?

**Comment by chimosky** [comment]
> I'm missing something. How is that different?

It's not different, skipped that in the list. Hiding comments as unrelated.
